pom.xml

```diff
@@ -1052,6 +1054,13 @@
                 <version>${swagger.models.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.wso2.carbon.analytics</groupId>
+                <artifactId>org.wso2.carbon.business.rules.core</artifactId>
+                <version>${carbon.analytics.version}</version>
+            </dependency>
+
+
```

Comment:
Unnecessary new lines

Reply:
This was removed and readded by 71a1fc6a259ce78d8aba76b1f4c0a93a5f54ddda